<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotFrameModifyUI</class>
 <widget class="QMainWindow" name="RobotFrameModifyUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1290</width>
    <height>860</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="VideoView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>49</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>1280</width>
      <height>720</height>
     </size>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="MotorFrame">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>560</y>
      <width>1231</width>
      <height>192</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="MotorMotion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>91</y>
      <width>291</width>
      <height>451</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::ClickFocus</enum>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1290</width>
     <height>28</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionFile_Open_2"/>
    <addaction name="actionFile_Save"/>
    <addaction name="actionFile_Save_as"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionFile_Open">
   <property name="text">
    <string>File Open</string>
   </property>
  </action>
  <action name="actionFile_Save">
   <property name="text">
    <string>File Save</string>
   </property>
  </action>
  <action name="actionFile_Save_as">
   <property name="text">
    <string>File Save as</string>
   </property>
  </action>
  <action name="actionFile_Open_2">
   <property name="text">
    <string>File Open</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
